### ISSUE TITLE
Received Promise is undefined in IE11

### ISSUE THREAD

**Issue by stanleyowen** [stanleyowen/otlio#204]
**Describe the bug**
While running Otlio on Internet Explorer 11 (`IE11`) it only returns a white blank page with the following error:
```
HTML1300: Navigation occurred.
login
SCRIPT5009: 'Promise' is undefined
asyncToGenerator.js (21,1)
SCRIPT5009: 'Promise' is undefined
index.js (1,1)
SCRIPT5009: 'Promise' is undefined
asyncToGenerator.js (21,1)
SCRIPT5009: 'Promise' is undefined
index.js (1,1)
```

**To Reproduce**
1. Go to Otlio Login Page through IE11
2. Open Developer Tools > Console
3. See error

**Expected behavior**
No `error` occurred and return `login` page instead of blank page.

**Screenshots**
![image](https://user-images.githubusercontent.com/69080584/121035856-7c777900-c7d8-11eb-907c-de7f5c8d5c3b.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 OS Version 2009 (Build 19043.1023)
 - Browser Chrome
 - Version 91.0.4472.77

**Comment by stanleyowen** [comment]
This issue will be part of #198 accomplishment because it is related to IE11.

**Comment by stanleyowen** [comment]
This issue has been fixed (0ec15a931952cea4b2d06ed51587b3708ddb1f9a) and will take effects on the next release.
